tests/python_flask_tests/config.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "0.1",
+    "Debug": 1,
+    "ProductID": 1,
+    "SecurityVersion": 1,
+    "MemorySize": "256m",
+    "CurrentWorkingDirectory": "/app/flask",
+    "ApplicationPath": "/usr/local/bin/python",
+    "ApplicationParameters": ["/usr/local/bin/pytest"],
+    "ForkMode": "pseudo_wait_for_exit_exec"
+}
```

Comment:
Pytest use multi-process. For Mystikos Multi-process support, "NoBrk" should be set (to true)

Reply:
Added "NoBrk": true but the test would still fail